app/onepassword_events_api/appserver/static/dashboard.css

```diff
@@ -0,0 +1,17 @@
+.dashboard-dashboardpage {
+  background: #0a2d4d;
+}
+
+.splunk-header > :nth-child(2),
```

Comment:
Are there classes or ids we can use over child selectors? If not disregard this comment, but typically child selectors are pretty brittle.

Reply:
Unfortunately there's only `.splunk-header / #header`. The bottom header has class `view---pages-enterprise---8-2-0---1zrJY` which I'm sure will change between different Splunk versions / installations etc.